Code/autopkglib/DmgMounter.py

```diff
@@ -120,21 +117,19 @@ def mount(self, pathname):
                     "-nobrowse",
                     pathname,
                 ),
-                stdout=subprocess.PIPE,
-                stderr=subprocess.PIPE,
-                stdin=subprocess.PIPE,
+                input=stdin,
+                capture_output=True,
                 text=True,
             )
-            (stdout, stderr) = proc.communicate(stdin)
         except OSError as err:
             raise ProcessorError(
                 f"hdiutil execution failed with error code {err.errno}: {err.strerror}"
             )
         if proc.returncode != 0:
-            raise ProcessorError(f"mounting {pathname} failed: {stderr}")
+            raise ProcessorError(f"mounting {pathname} failed: {proc.stderr}")
```

Comment:
Same feedback here as above - we should leverage check=True.

Reply:
e376acd